packages/flutter_tts/tizen/src/text_to_speech.h

```diff
@@ -11,30 +11,24 @@
 #include <string>
 #include <vector>
 
+enum class TtsState { kCreated, kReady, kPlaying, kPaused, kError };
```

Comment:
How about this?
```suggestion
enum class TtsState {
  kCreated = TTS_STATE_CREATED,
  kReady = TTS_STATE_READY,
  kPlaying = TTS_STATE_PLAYING,
  kPaused = TTS_STATE_PAUSED,
};
```
then I guess you can use like this
``` c++
  tts_state_e state;
  int ret = tts_get_state(tts_, &state);
  TtsState tts_state(state);
```